articles/advisor/advisor-reference-cost-recommendations.md

```diff
@@ -488,10 +488,7 @@ Learn more about [App Service plans](../app-service/overview-hosting-plans.md).
 
 ### Unused/Empty App Service plans
 
-Your App Service plan has no apps running for at least three days. Consider deleting the resource to save costs or add new apps under it.
-
-> [!NOTE]
-> It might take up to 48 hours for this recommendation to refresh after you take an action.
+The App Service plan you have does not have any running applications associated with it. You may want to delete the resource to reduce expenses or add new apps under it to utilize its capabilities.
```

Comment:
Your App Service plan does not have any running applications associated with it. Consider deleting the resource to reduce expenses, or add new applications under it to utilize its capabilities.

Reply:
Got it. The most crucial point was to remove the 3-days delay, but I thought it would be nice to rephrase it. Glad you gave your comment. Just fixed it!